packages/gatsby/src/commands/serve.js

```diff
@@ -61,6 +61,38 @@ const matchPathRouter = (matchPaths, options) => (req, res, next) => {
   return next()
 }
 
+const readRedirects = async program => {
+  const filePath = path.join(program.directory, `.cache`, `redirects.json`)
+  let rawJSON = `[]`
+  try {
+    rawJSON = await fs.readFile(filePath)
```

Comment:
Can't we require the file instead of reading it and then parsing it?

Reply:
Good point! I followed the same pattern that was already in use for reading `match-paths.json`, but I'm happy to change them both over to require if you'd like. 

Do you think there'd be any issues with require if the file is created/updated after serve is first run, or while it's running?

Reply:
Hey @sidharthachatterjee - any thoughts on the above?